### DISCUSSION TITLE
'next' is not recognized as an internal or external command, operable program or batch file.

### DISCUSSION THREAD

**Discussion by JamesSingleton** [vercel/next.js#26638] [Help] [answered]
Is anyone else getting `'next' is not recognized as an internal or external command, operable program or batch file.` on Windows lately? I have one [repo](https://github.com/JamesSingleton/investead) that is on 10.2.3 that I can run `yarn dev` on just fine. However, creating a new Next.js app and running `yarn dev` results in that error on Windows, even if I downgrade Next to 10.2.3 it still happens.

![yarn_dev_w_next](https://user-images.githubusercontent.com/21000200/123563401-4be88680-d769-11eb-999a-4be346fe547b.gif)

**Comment by JamesSingleton** [comment, marked as answer]
For some reason upgrading to Yarn 2 fixed the issue I was having.

**Comment by vierynugroho** [comment]
just make sure that there are no folders named with numbers in front and unique characters
this solved my problem
